main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/companieshouse/chs.go/avro"
+	"github.com/companieshouse/chs.go/avro/schema"
+	"github.com/companieshouse/chs.go/kafka/consumer/cluster"
+	"github.com/companieshouse/chs.go/kafka/producer"
+	"github.com/companieshouse/chs.go/kafka/resilience"
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/company-search-consumer/config"
+	"github.com/companieshouse/company-search-consumer/service"
+	"github.com/companieshouse/company-search-consumer/upsert"
+	gologger "log"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	cfg := config.Get()
+
+	log.Namespace = "company-search-consumer"
+	log.Trace("company-search-consumer starting...", log.Data{"config": cfg})
+
+	// Push the Sarama logs into our custom writer
+	sarama.Logger = gologger.New(&log.Writer{}, "[Sarama] ", gologger.LstdFlags)
+
+	var resetOffset bool
+	if cfg.InitialOffset != -1 {
+		resetOffset = true
+	}
+
+	resourceChangedDataSchema, err := schema.Get(cfg.SchemaRegistryURL, "resource-changed-data")
+	if err != nil {
+		log.Error(err)
+		os.Exit(1)
+	}
+
+	p, err := producer.New(&producer.Config{Acks: &producer.WaitForAll, BrokerAddrs: cfg.StreamingBrokerAddr})
+
+	retry := &resilience.ServiceRetry{
+		time.Duration(cfg.RetryThrottleRate),
+		cfg.MaxRetryAttempts,
+	}
```

Comment:
Keys should be named
```suggestion
      retry := &resilience.ServiceRetry{
		ThrottleRate: time.Duration(cfg.RetryThrottleRate),
		MaxRetries:   cfg.MaxRetryAttempts,
	}
```